content/english/blog/2024-06-open-sauce.md

```diff
@@ -0,0 +1,172 @@
+off that, but I took at least a little comfort in some industry knowledge about these things - these big generators are
+really just small, 3-phase power plants, frequently used to power expensive concert equpiment without incident - lighting,
+sound boards, speakers, extremely rare and expensive vintage synthesizers, the whole nine-yards.
+
+Unfortunately all of that
+stability comes to a deafening fizzle thanks to nature's tendency to invent idiots who suddenly believe they're qualified
+to go throwing switches on the generator's control panel. Just a few seconds into that final game, a brown out due to
+improper generator operation sent the arm crashing into the target controller on its back, and the address decoding
+circuit in the memory controller to send a logfile write, which normally goes to battery-backed RAM, destined instead
+to to the flash chips storing unreplaceable firmware and calibration values.
+
+<img class="img-responsive" src="/images/blog/2024/open-sauce/broken-bot-6.jpg" title="The robot controller's console now only boots to immediately kernel panic">
+
+
+Last year in the course of learning more about our controller, I'd attempted to take a backup of the internal flash
+on the controller, but after revisiting those backups I'm totally disheartened to learn that 1 of the 4 flash chip dumps
+is empty.
+
+If you know anyone with this era of robot arm controller, that could be instrumental in getting our project working
+again, please drop us an email here at info [at] pawprintprototyping.org.
+
+
+## Pirate Box Badge
+
+<img class="img-responsive" src="/images/blog/2024/open-sauce/pirate-box.jpg" title="PirateFox Badge, the Wifi router and SneakerNet dead drop you can wear around your neck">
+
+This [#BadgeLife](https://defcon.social/tags/badgelife) entry for DEF CON, the world's largest hacking conference, was
+designed by Geo and hosts a wide variety of computer networking abilities.  It's also a cute fox with zero depth-perception.
+
+
+## Long Range Thermal Imaging
+
+<img class="img-responsive" src="/images/blog/2024/open-sauce/thermal-camera.jpg" title="This ITAR-restricted bundle of joy can see hot things from very far away">
+<img class="img-responsive" src="/images/blog/2024/open-sauce/thermal-camera-2.jpg" title="A transparent look at the fursuiting experience: yes, it's hot in there, and no, we can't see a thing in any spectrum.">
+
+This thermal camera, built to be mounted to the outside of a building for surveillance purposes, was reverse engineered
+by Geo to be controlled by a playstation game controller.  Are you hot or not?  Finally, science has the answer.
+
+## Water Plants Game
+
+<img class="img-responsive" src="/images/blog/2024/open-sauce/water-plants.png" title="Water plants watering dude">
+
+In this game by Pawprint member Dingo, help our friend get the chores of watering the plants around the water plant so
+they can get back to their favorite activity: napping.
```

Comment:
![image](https://github.com/PawprintPrototyping/pawprintprototyping.org-hugo/assets/1631158/12d4746e-bd55-4d12-b194-a647072742ef)
![image](https://github.com/PawprintPrototyping/pawprintprototyping.org-hugo/assets/1631158/1c2794ca-a082-47ee-aac8-29c3ac9080d3)
![image](https://github.com/PawprintPrototyping/pawprintprototyping.org-hugo/assets/1631158/24ea2820-4b26-4d5d-b6cd-ac74b35910ac)

Here's some full screenshots from the [itch.io page](https://doomsdaydingo.itch.io/water-plants)